app/js/MongoManager.js

```diff
@@ -59,7 +59,9 @@ module.exports = MongoManager = {
     }
     db.docs
       .find(query, {
-        projection: filter
+        projection: filter,
+        // use a specific index
+        plan: 'project_id_1'
```

Comment:
I can't see a `plan` option on http://mongodb.github.io/node-mongodb-native/3.6/api/Collection.html#find  (the nearest one is `hint`).

Reply:
I see you mentioned it in the PR description, so maybe `plan` is a typo?

Reply:
Right, I even quotes that in the PR description :facepalm:

Reply:
Fixed via force-push: https://github.com/overleaf/docstore/compare/ac1f9ded6362f45991f38b1b34fe391b67ed6b3d..56c780f51783405882d03b6c978cfe0c66090a24

Thanks for spotting that!

Reply:
CI is failing now. We will need to open source the indexes and add them during the acceptance test run.